src/codeu/chat/util/Time.java

```diff
@@ -62,6 +62,11 @@ public String toString() {
 		return formatter.format(date);
 	}
 
+	//time in just hours and minutes
+  	public final String HMtime() {
+  		return formatter.format(date).substring(12,17);
```

Comment:
Since the date is always associated with a user/conversation/message, it should always have a value. I'll add a try catch statement just in case though. Good catch!